app/javascript/controllers/flatpickr_controller.js

```diff
@@ -0,0 +1,5 @@
+import { Application } from '@hotwired/stimulus'
+import Flatpickr from 'stimulus-flatpickr'
```

Comment:
The tailwinds component implementation offers this same functionality. https://flowbite.com/docs/plugins/datepicker/

Reply:
But you still need to install flowbite and then add datepicker.js.
Should we explore that and do it in a different ticket? 
It's not essential for this PR how a calendar is displayed in a date field. I did the same thing we did in AVM.

Reply:
@britaumich I wanted to be sure you were aware of this library